public/locales/id/translation.json

```diff
@@ -0,0 +1,64 @@
+{
+    "about": "Tentang Kami",
+    "available": "Tersedia",
+    "blog": "Pengenalan",
+    "community": "Komunitas",
+    "contributors": "Kontributor",
+    "countryCode": "id",
+    "title": "Buat nama proyekmu jadi lebih keren",
+    "description": "Cek ketersediaan nama aplikasi dan organisasimu di berbagai platform dengan mudah.",
```

Comment:
Though I'm new to Indonesian, I think `dan` should've been `atau`. Or did I miss something?

Reply:
@LukasPurbaW /ping

Reply:
sorry for the late reply, yes in term of meaning `atau` is right, but the word itself is quite hard to say, that's why I use `dan`. Personally I like to use `dan` because it's easier to say (only one syllable) and it doesn't really change the true meaning of the sentence.

Reply:
I got it now. Thanks for the clarification!